### PR TITLE
OCPBUGS-31694: E2E: Workload hints test cases fixes 

### PR DESCRIPTION
**Pull request by mrniranjan**

### CODE REVIEW

File: test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go [hunk 1]
```diff
@@ -258,28 +259,32 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 
 		When("realtime and high power consumption enabled", func() {
 			It("[test_id:50993][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
+				currentWorkloadHints := profile.Spec.WorkloadHints
```
Comment (ffromani) [inline test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go:262]: do we need a DeepCopy() here?

File: test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go [hunk 2]
```diff
@@ -320,27 +325,29 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 
 		When("perPodPowerManagent enabled", func() {
 			It("[test_id:54177]should update kernel arguments and tuned accordingly", func() {
+				currentWorkloadHints := profile.Spec.WorkloadHints
```
Comment (ffromani) [inline test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go:328]: ditto: do we need a DeepCopy() here?

**Review by ffromani — COMMENTED** [review]
/approve

LGTM once a inline question is addressed

File: test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go (hunk 1 above)
Comment (mrniranjan) [reply test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go:262]: You mean should we DeepCopy() ?

Comment (mrniranjan) [reply test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go:262]: Here, i only intend to get the current profile before i modify, to check if the current profile is already contains the changes i intend to make further, if the current profile already contains the changes needed,  then don't need to apply the spec changes , directly check the tuning parameters.

**Review by ffromani — COMMENTED** [review]
/lgtm

**Pull request merged**